### PR TITLE
Fix error when getting means for course without aggregators

### PR DESCRIPTION
**Pull request by xitij2000**
**Description:** 

Fixes error that occurs when you ask for mean value in a course without any Aggregators. 

**Reviewers:**
- [ ] @jcdyer 

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Version bumped
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed
- [ ] PR author is listed in AUTHORS

**Post merge:**
- [ ] Create a tag
- [ ] Check new version is pushed to PyPI after tag-triggered build is 
      finished.
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

File: completion_aggregator/serializers.py [hunk 1]
```diff
@@ -259,22 +259,32 @@ def get_username(self, obj):
...
+        """
+        enrollments = compat.get_users_enrolled_in(obj.course_key)
+        enrollment_count = enrollments.count()
+        if enrollment_count == 0:
+            return 0.
+
+        total = Aggregator.objects.filter(
+            course_key=obj.course_key,
+            aggregation_name='course',
+        ).aggregate(Sum('percent')).get('percent__sum', 0.)
```
Comment (jcdyer) [inline completion_aggregator/serializers.py]: I'm not sure this is right.  I think if there are any nulls, the entire sum will become 0.  I suspect we need a more complex sql expression for the aggregate? Something like `Sum(Coalesce('percent', Value(0)))`.

[Database function documentation](https://docs.djangoproject.com/en/2.1/ref/models/database-functions/)

Comment (xitij2000) [reply completion_aggregator/serializers.py]: OK! Didn't know that. Will update the PR and add a test for this.

Comment (xitij2000) [reply completion_aggregator/serializers.py]: I've updated the code, but I don't think it's possible for percent to be null, so I'm not sure it's possible to test this.

File: completion_aggregator/serializers.py [hunk 2]
```diff
@@ -271,10 +272,10 @@ def _calculate_mean(self, obj):
         total = Aggregator.objects.filter(
             course_key=obj.course_key,
             aggregation_name='course',
-        ).aggregate(Sum('percent')).get('percent__sum', 0.)
+        ).aggregate(
+            total=Sum(Coalesce('percent', Value(0.)))
+        ).get('total') or 0.
```
Comment (jcdyer) [inline completion_aggregator/serializers.py]: Looks cleaner too.

**Pull request merged**